frontend/src/metabase/admin/upsells/components/UpsellBanner.tsx

```diff
@@ -1,12 +1,11 @@
 import { useMount } from "react-use";
 
+import ExternalLink from "metabase/core/components/ExternalLink";
```

Comment:
Do we want to continue proliferating ExternalLink or should we be using Mantine's anchor component instead? I'd added ExternalLink to the `Ask before using` folder in my storybook cleanup PR because I think it could use a rework to either use Mantine anchor under the hood, or could potentially be replaced.

Reply:
ExternalLink is doing some extra stuff for us, I think we should keep using it, but maybe make _it_ use Anchor under the hood.